tools/devtool

```diff
@@ -822,11 +822,21 @@ mount_ramdisk() {
     umount_ramdisk
     mkdir -p ${DEFAULT_RAMDISK_PATH} && \
     mount -t tmpfs -o size=${ramdisk_size} tmpfs ${DEFAULT_RAMDISK_PATH}
+    ok_or_die "Failed to mount ramdisk to ${DEFAULT_RAMDISK_PATH}. Check the permission."
     mkdir -p ${DEFAULT_RAMDISK_PATH}/srv
     mkdir -p  ${DEFAULT_RAMDISK_PATH}/tmp
 }
 
 umount_ramdisk() {
+    if [ ! -e "${DEFAULT_RAMDISK_PATH}" ]; then
+        return 0
+    fi
+    if [ ! -d "${DEFAULT_RAMDISK_PATH}" ]; then
+        die "${DEFAULT_RAMDISK_PATH} is not a directory."
+    fi
+    if [ ! -w "${DEFAULT_RAMDISK_PATH}" ]; then
+        die "Failed to unmount ${DEFAULT_RAMDISK_PATH}. Check the permission."
+    fi
     umount ${DEFAULT_RAMDISK_PATH} &>/dev/null
```

Comment:
I suggest we simply do a check with `ok_or_die` after the call to `umount`, just like we do with `mount`

Reply:
It might not be a good idea. If the directory does not exist, `umount` returns non-zero, and if the directory is not mounted, `umount` returns non-zero as well.
`umount_ramdisk` must have idempotency, so I didn't use `ok_or_die`.

Reply:
makes sense